src/metabase/driver/sql_jdbc.clj

```diff
@@ -170,8 +168,9 @@
         ;; across all drivers. With that in mind, 100 seems like a safe compromise.
         ;; There's nothing magic about 100, but it felt good in testing. There could well be a better number.
         chunks     (partition-all (or driver/*insert-chunk-rows* 100) values)
-        sqls       (map #(sql/format {:insert-into table-name
-                                      :columns     columns
+        sqls       (map #(sql/format {:insert-into (keyword table-name)
+                                      ;; We need to namespace the keyword in case the column name starts with a %
+                                      :columns     (map (comp (partial keyword table-name) name) column-names)
```

Comment:
uhhh that `comp` (and the one in postgres.clj) is way harder to read compared to the `#(...)` version in mysql.clj